torchci/pages/metrics.tsx

```diff
@@ -542,16 +542,44 @@ export default function Page() {
           >
             <ScalarPanel
               title={"% force merges due to failed PR checks"}
-              queryName={"force_merge_red_avg"}
-              metricName={"force_merges_red"}
-              valueRenderer={(value) => (value * 100).toFixed(1) + "%"}
-              queryParams={timeParams}
+              queryCollection={"commons"}
+              queryName={"weekly_force_merge_stats"}
+              metricName={"metric"}
+              valueRenderer={(value) => (value).toFixed(1) + "%"}
+              queryParams={[
+                {
+                  name: "one_bucket",
+                  type: "bool",
+                  value: "True",
+                },
+                {
+                  name: "merge_type",
+                  type: "string",
+                  value: "Failure",
+                },
+                ...timeParams,
+              ]}
               badThreshold={(value) => value > 0.055}
```

Comment:
As this is already in %, the bad threshold here can be updated to 5.5.  Do you know why it's 5.5 instead of a round 5.0 number?

Reply:
Good catch on the bug!
I used decimal places because the numbers are getting small enough that it could materially affect our KPI.  That said, this number was actually wrong. Fixed now